### DISCUSSION TITLE
What's the right way to suggest devops related RFE

### DISCUSSION THREAD

**Discussion by dpkshetty** [github/india#110] [Satellite India 2021 Q&A]
Hi, I have an idea/suggestion for a new feature for Github which can help improve devops workflow / efficiency.
What is the best way to suggest an RFE for github ?
Is there a online form/page where I can document the new RFE such that I get the credit (if it turns into a feature in github) ?
Let me know

**Comment by FRTMentor** [comment]
Please reach out the GitHub support team.
